tests/ra_aid/agents/test_ciayn_agent.py

```
@@ -154,3 +155,49 @@ def test_trim_chat_history_both_limits():
+
+
+class TestFunctionCallValidation:
+    @pytest.mark.parametrize("test_input", [
+        "basic_func()", 
+        "func_with_arg(\"test\")",
+        "complex_func(1, \"two\", three)",
+        "nested_parens(func(\"test\"))",
+        "under_score()",
+        "with-dash()"
+    ])
+    def test_valid_function_calls(self, test_input):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_valid_function_calls`

Add return type annotation: `None`

(ANN201)